commands/build.go

```diff
@@ -304,20 +305,19 @@ func build(services *stack.Services, queueDepth int, shrinkwrap, quietBuild bool
 }
 
 // pullTemplates pulls templates from specified git remote. templateURL may be a pinned repository.
-func pullTemplates(templateURL string) error {
-	var err error
-	exists, err := os.Stat("./template")
-	if err != nil || exists == nil {
-		log.Println("No templates found in current directory.")
+func pullTemplates(templateURL, templateName string) error {
+
+	if _, err := os.Stat("./template"); err != nil && os.IsNotExist(err) {
```

Comment:
More idiomatic way to check for the folder.

Reply:
Agree.  

Would `os.Stat` return any errors other than  `os.IsNotExist`?